portal/src/graphql/portal/BotProtectionConfigurationScreen.tsx

```diff
@@ -180,13 +181,28 @@ const BotProtectionConfigurationContent: React.VFC<BotProtectionConfigurationCon
   function BotProtectionConfigurationContent(props) {
     const { form } = props;
     const { state, setState } = form;
+    const [storedFormState, setStoredFormState, removeStoredFormState] = useLocalStorage<FormState>(
```

Comment:
Lets use [sessionStroage](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage) instead of localStorage, to ensure the state will not persist after the user closed the browser.